relayer/chain/ethereum/connection.go

```diff
@@ -57,3 +57,7 @@ func (co *Connection) Close() {
 func (co *Connection) GetClient() *ethclient.Client {
 	return co.client
 }
+
+func (co *Connection) GetKeyPair() *secp256k1.Keypair {
```

Comment:
seems like elsewhere we just use wr.conn.kp directly - can we be consistent, ie, just remove this and use it directly, or use the GetKeyPair() function throughout the writer?

Reply:
This is orphaned code that was previously used in a test file. Will remove.

Reply:
Removed in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/bc765754f8d0e0df068a2b39d0e6bf1225b6eb18